App.tsx

```diff
@@ -14,10 +14,17 @@ const CodeInput = () => {
   const [code, setCode] = useState('');
   const [containerIsFocused, setContainerIsFocused] = useState(false);
 
-  const codeDigitsArray = new Array(CODE_LENGTH);
+  const codeDigitsArray = [...Array(CODE_LENGTH)];
```

Comment:
use spread operator to create array of CODE_LENGTH length, the original code could not be mapped

Reply:
This fixed my issue as well. Otherwise, the fields do not show.